spec/requests/users_controller_spec.rb

```diff
@@ -73,11 +73,11 @@
     end
 
     describe "title link" do
-      it "routes user to the /logs page" do
+      it "routes user to the home page" do
         sign_in user
         get "/", headers:, params: {}
-        follow_redirect!
-        expect(path).to include("/lettings-logs")
+        expect(path).to include("/")
```

Comment:
could this be to_eq? Or `expect(path).not_to include("/lettings-logs")`, cause otherwise we might not be testing very much here